pandas/core/arrays/interval.py

```diff
@@ -146,7 +146,7 @@
     >>> pd.arrays.IntervalArray([pd.Interval(0, 1), pd.Interval(1, 5)])
     <IntervalArray>
     [(0, 1], (1, 5]]
-    Length: 2, closed: right, dtype: interval[int64]
+    Length: 2, closed: right, dtype: interval[int64, right]
```

Comment:
should prob change the output formatting as well (e.g. closed: right) is now superfluous

Reply:
agreed